### PR TITLE
refactor(loadbalancer): migrate manual_certificate_bundle to plugin fw

### PR DESCRIPTION
**Pull request by villevsv-upcloud**

### CODE REVIEW

File: internal/service/loadbalancer/manual_certificate_bundle.go [hunk 1]
```diff
@@ -5,157 +5,219 @@ import (
...
+			"certificate": schema.StringAttribute{
+				MarkdownDescription: "Certificate within base64 string must be in PEM format.",
+				Required:            true,
 			},
-			"certificate": {
-				Description: "Certificate within base64 string must be in PEM format.",
-				Type:        schema.TypeString,
-				Required:    true,
+			"id": schema.StringAttribute{
+				MarkdownDescription: "Computed ID of the resource.",
```
Comment (kangasta) [inline internal/service/loadbalancer/manual_certificate_bundle.go:66]: Could clarify that UUID is used as the ID:

```suggestion
				MarkdownDescription: "The UUID of the certificate bundle",
```

File: internal/service/loadbalancer/manual_certificate_bundle.go [hunk 2]
```diff
@@ -5,157 +5,219 @@ import (
...
+				Optional:            true,
+				Computed:            true,
 			},
-			"private_key": {
-				Description: "Private key within base64 string must be in PEM format.",
-				Type:        schema.TypeString,
-				Required:    true,
-				Sensitive:   true,
+			"name": schema.StringAttribute{
+				MarkdownDescription: "The name of the bundle must be unique within customer account.",
```
Comment (kangasta) [inline internal/service/loadbalancer/manual_certificate_bundle.go:78]: Could split these descriptions to make them more clear, but not sure if it makes sense to do that in API docs first. E.g.:

```suggestion
				MarkdownDescription: "The name of the certificate bundle. Must be unique within customer account.",
```

**Review by kangasta — APPROVED** [review]
LGTM 👌 Added few comments about the descriptions.

File: internal/service/loadbalancer/manual_certificate_bundle.go [hunk 3]
```diff
@@ -5,157 +5,219 @@ import (
...
+	OperationalState types.String `tfsdk:"operational_state"`
+	PrivateKey       types.String `tfsdk:"private_key"`
+}
+
+func (r *manualCertificateBundleResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		MarkdownDescription: "This resource represents manual certificate bundle",
+		Attributes: map[string]schema.Attribute{
+			"certificate": schema.StringAttribute{
+				MarkdownDescription: "Certificate within base64 string must be in PEM format.",
```
Comment (kangasta) [inline internal/service/loadbalancer/manual_certificate_bundle.go:62]: Would this be easier to understand? (Same issue also in API docs and `intermediates`)

```suggestion
				MarkdownDescription: "Certificate as base64 encoded string. Must be in PEM format.",
```

File: internal/service/loadbalancer/manual_certificate_bundle.go (hunk 2 above)
Comment (villevsv-upcloud) [reply internal/service/loadbalancer/manual_certificate_bundle.go]: These descriptions have now been split

File: internal/service/loadbalancer/manual_certificate_bundle.go (hunk 3 above)
Comment (villevsv-upcloud) [reply internal/service/loadbalancer/manual_certificate_bundle.go]: Changed the description(s) accordingly

**Pull request merged**